app/javascript/modules/datadog.js

```diff
@@ -0,0 +1,20 @@
+import { datadogLogs } from '@datadog/browser-logs'
+
+const initDatadog = () => {
+    const ddEnv = window.location.hostname === 'www.citizensadvice.org.uk' ? "prod" : "dev";
+    const appVersion = document.querySelector('meta[name="app-version"]').getAttribute('content');
+
+    if (ddEnv && appVersion) {
+        datadogLogs.init({
+            clientToken: 'pub32a7538113d7910e4304b8aa13ff4521',
```

Comment:
Could you add a comment explaining that this is designed to be a public token - it's not a security issue to have this in the code?

Are client tokens linked domain name or something? What stops someone from spamming our logs? 🤔

Reply:
Yep I can add a comment.

It looks like datadog are aware this could be used to send spam data - https://github.com/DataDog/browser-sdk/issues/853 and https://github.com/DataDog/browser-sdk/issues/853#issuecomment-843153872

I'm not sure there is a way around this really? Could we build smth that forwards errors to our own endpoint and maybe use the DD API to create logs? Then we'd have more control over what ends up in the logs and from where, but I'm not sure it's worth the effort?

Reply:
Discussed on Slack 👍